Makefile.devel

```diff
@@ -362,12 +345,13 @@ cvsexport:
 	rm -rf $(DISTBUILDIR)
 	mkdir -p $(DISTBUILDIR)
 	if [ -z "$(NOCVS)" ]; then \
-           (cd $(DISTBUILDIR); \
-	    cvs -d $(CVSROOT) export -kv -r "${CVS_RELEASENAME}" -d ${RELEASENAME} ${CVSNAME}) \
+	    git worktree prune; \
```

Comment:
`git worktree` is definitely overkill here.
Instead, you could replace these two lines with
`git archive --format=tar --prefix=$(RELEASENAME)/ $(GIT_RELEASENAME) | (cd "$(DISTBUILDIR)" && tar xf -)`
**and** remove the `-rm -rf "$(DISTBUILDIR)/$(RELEASENAME)/.git"` line below.